pgbouncer/values.yaml

```diff
@@ -1,39 +1,57 @@
+# For all pgbouncer config, please check https://www.pgbouncer.org/config.html
 # Deployment values for pgbouncer
 replicaCount: 1
 internalPort: 5432
 servicePort: 5432
 antiAffinity: soft
 nodeAffinity: {} # optionally define nodeAffinity
 tolerations: [] # optionally define tolerations
-nodeSelector: {} # optionally define nodeSelector
```

Comment:
why was this removed?

Reply:
sorry for this, I removed it as nodeSelector will be deprecated, as node affinity can express everything that nodeSelector can express.